nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# a minimal Nginx container including containerbuddy and a simple virtualhost config
```

Comment:
I'd do it exactly as I've done here -- each app managed by the compose file in its own directory.